vip_spec.xsd

```diff
@@ -8,6 +8,7 @@ reference elements for polling location
   <xs:element name="VipObject">
     <xs:complexType>
       <xs:choice maxOccurs="unbounded">
+
```

Comment:
I added the lines around the 'choice' tags because the end of the file was kind of a mess. The other blank lines I added were to create consistency with the other definitions which had blank lines between them. Although yes, we should add blank line use to the style guide.